examples/spring-boot/src/test/java/com/example/DemoControllerTest.java

```diff
@@ -1,16 +1,20 @@
 package com.example;
 
-import static org.rnorth.visibleassertions.VisibleAssertions.*;
-
 import org.junit.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.web.client.TestRestTemplate;
 
+import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
```

Comment:
For a general purpose example, I'd avoid using assertion methods from a third-party library. Those imports should come from JUnit e.g. `org.junit.Assert. assertEquals`. Users will likely ask themselves which of the dependencies brings in the import. The `info` method could probably be replaced by something else.